pkg/cvo/metrics.go

```diff
@@ -424,6 +520,13 @@ func gaugeFromInstallConfigMap(cm *corev1.ConfigMap, gauge *prometheus.GaugeVec,
 	return g
 }
 
+func logCurrentAndReturnNewError(currentError, newError error) error {
```

Comment:
[lint][1]:

```
pkg/cvo/metrics.go:523:6  deadcode  `logCurrentAndReturnNewError` is unused
```

Can we drop this function?

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/726/pull-ci-openshift-cluster-version-operator-master-golangci-lint/1483827196928724992#1:build-log.txt%3A16

Reply:
As of 0e6136c727b055, `logCurrentAndReturnNewError` is gone.